.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
+}
```

Comment:
Why/how did this end up being added? :)

Reply:
removing this via https://github.com/opstrace/opstrace/pull/718 -- let's have discussion over there if desired

Reply:
the answer is: mistakenly :) Thanks @jgehrcke 

Can we hold off merging this till after we merge @terrcin's PRs? I think he also removed it and his PRs are ready to go. I'd like to land them and asap!

Reply:
Good catch! https://github.com/opstrace/opstrace/pull/701#discussion_r631065603